packages/functions/src/coordinator/GetCoordinatorAppointmentsHandler.ts

```diff
@@ -71,22 +71,33 @@ async function fetchCoordinator(callerId: string) {
   return coordinator;
 }
 
-async function validate(coordinator: DbCoordinator, hospital: Hospital) {
+async function validate(
+  coordinator: DbCoordinator,
+  hospital: Hospital | "all"
+): Promise<Hospital[]> {
   switch (coordinator.role) {
     case CoordinatorRole.SYSTEM_USER:
-      return;
     case CoordinatorRole.ZM_COORDINATOR:
     case CoordinatorRole.HOSPITAL_COORDINATOR:
+      if (hospital === "all") {
```

Comment:
For SYSTEM_USER and ZM_COORDINATOR you should return all active hospitals

Reply:
it returns the hospital you ask for or all hospitals if ask for "all".. (see lines 96-100)

Reply:
fixed

Reply:
I don't see the fix, where is it?

Reply:
it passes the switch case and returns the ospitals that were asked or all the active hospitals

Reply:
Are you sure?

Anyway, make sure you have tests that cover this